App.js

```diff
@@ -180,30 +189,34 @@ export default class App extends React.Component {
       activateKeepAwake()
 
       return (
-        <Provider store={store}>
-          <View style={styles.mainContainer}>
-            <StatusBar
-              translucent={true}
-              hidden={false}
-              barStyle="dark-content"
-            />
-            <LocaleContext.Provider value={this.state}>
-              <AppNavigator
-                ref={navigatorRef => {
-                  NavigationService.setTopLevelNavigator(navigatorRef);
-                }}
-                screenProps={{
-                  t: this.t,
-                  locale: this.state.locale,
-                  localize: this.localize,
-                  changeLanguage: this.changeLanguage
-                }}
+        <ThemeContext.Provider value={this.state}>
+          <Provider store={store}>
+            <View style={[styles.mainContainer, { backgroundColor: this.state.theme.background }]}>
+              <StatusBar
+                translucent={true}
+                hidden={false}
+                barStyle="dark-content"
               />
-            </LocaleContext.Provider>
-            {/*https://www.npmjs.com/package/react-native-flash-message?activeTab=readme*/}
-            <FlashMessage position="bottom" />
-          </View>
-        </Provider>
+              <LocaleContext.Provider value={this.state}>
+                <AppNavigator
+                  ref={navigatorRef => {
+                    NavigationService.setTopLevelNavigator(navigatorRef);
+                  }}
+                  screenProps={{
+                    t: this.t,
+                    locale: this.state.locale,
+                    localize: this.localize,
+                    changeLanguage: this.changeLanguage,
+                    themeBg: this.state.theme.background
+                  }}
+                />
+              </LocaleContext.Provider>
+              <ThemeTogglerButton />
```

Comment:
I like this toggle button. I think this button can be put into AccountScreen instead of on every screen.